modernize/__main__.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import generator_stop
```

Comment:
@brettcannon I'm not really sure what this file was doing outside of a package. It didn't seem to work, so I moved it in here where it does